values.yaml

```
@@ -222,25 +238,38 @@ global:
         memory: "50Mi"
         cpu: "20m"
 
-  # imageEnvoy is the name (and tag) of the Envoy Docker image used for the
-  # connect-injected sidecar proxies and the mesh, terminating, and ingress gateways.
+  # The name (and tag) of the Envoy Docker image used for the
+  # connect-injected sidecar proxies and mesh, terminating, and ingress gateways.
   # See https://www.consul.io/docs/connect/proxies/envoy for full compatibility matrix between Consul and Envoy.
   imageEnvoy: "envoyproxy/envoy-alpine:v1.16.0"
 
-  # openshift contains configuration for running this Helm chart
-  # on the Red Hat OpenShift platform.
+  # Configuration for running this Helm chart on the Red Hat OpenShift platform.
   # This Helm chart currently supports OpenShift v4.x+.
   openshift:
-    # If true, the Helm chart will create necessary configuration for running its components
-    # on OpenShift.
+    # If true, the Helm chart will create necessary configuration for running
+    # its components on OpenShift.
     enabled: false
 
 # Server, when enabled, configures a server cluster to run. This should
 # be disabled if you plan on connecting to a Consul cluster external to
 # the Kube cluster.
 server:
+
+  # If true, the chart will install all the resources necessary for a
+  # Consul server cluster. If you're running Consul externally and want agents
+  # within Kubernetes to join that cluster, this should probably be false.
+  # @default: global.enabled
+  # @type: boolean
   enabled: "-"
+
+  # The name of the Docker image (including any tag) for the containers running
+  # Consul server agents.
+  # @type: string
   image: null
+
+  # The number of server agents to run. This determines the fault tolerance of
+  # the cluster. Please see the deployment table (https://consul.io/docs/internals/consensus#deployment-table)
+  # for more information.
   replicas: 3
 
   # bootstrapExpect is the number of servers that are expected to be running.
```

```suggestion
  # The number of servers that are expected to be running.
```
This looked like the kind of change that was being made everywhere, feel free to ignore if this is unnecessary here.

---

No good catch, this was still here because I rebased off of master and this was a new doc block that was written in the old style.